src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -6,6 +6,126 @@ description: Debriefs and takeaways from our performances at RoboCup.
+- NUC died due to overheating
+- Needed more monitors: one was just not enough
+- Issues of wiring getting in the way – makes it hard to swap things out
+- No soldering happened, but the setup could have been better if we did use it
+- Long ethernet cables became tedious to use and untangle
+- Heat dissipation is an issue
+
+#### Networking
+
+- Issues with connecting to GameController during games
+
+#### Vision
+
+- Red camera. Possible hardware issue? Swapping the cable fixed the issue, but this may have reset the automatic white balance
+- New mesh is too computationally expensive for the NUCs that we have. Potentially go down to one camera per robot and lower the number of images received from the camera
+- The bottom of the mesh, near the robot's feet, bends around, possibly causing issues with ball detection near the feet
+- Hoc was slightly off - odometry or camera calibration
+- NUsight was unreliable for testing vision
+- Theta was in check-in, not carry-on, so couldn’t collect data until later. Did get one image earlier using a theta from CIT Brains
+
+#### NUsight
+
+- Difficulty connecting to NUsight
+
+#### Localisation and Odometry
+
+- No localisation
+- Hoc was slightly off for vision
+
+#### Miscellanious
+
+- No robots until end of first competition day due to missing luggage
+
+### What went well
+
+#### Hardware
+
+- Didn’t replace a single servo at the competition
+- Padding was effective for keeping the robot protected and for colour requirement in inspection
+
+#### Motions
+
+- The robots were able to walk stabily!!
+- Getups basically worked flawlessly
+
+#### Networking
+
+- Able to connect to GameController for most of the games
+
+#### Odometry
+
+- Pitch and roll were good, better than the previous year
+
+#### Vision
+
+- Fine tuning for vision worked well despite not having our own Theta camera
+
+#### Teamwork and Moral
+
+- Got robot ready for qualification very quickly
+- We all came to the games - had three different robot handlers for four games
+- Got lots of promotional footage of the robot
+- We made the quarter finals despite not having two whole set up days and the first competition day to test the system and networking
+- Within 30 minutes of getting our robots from the airport, the robots were walking and getting up
+- Good preparation before the competition was vital to the team being able to perform at all when the robots arrived at the end of the first competition day
+- Overall made good progress compared to RoboCup 2019 Sydney
+- On par or better than Team-KMUTT, UTRA, Hamburg Bit-Bots, Bold Hearts, ITAndroids, RoboFEI
+- Students at home were around to help with vision fine-tuning
+
+### What’s Next
+
+#### Networking
+
+- System configuration tool
+- Configurable headless mode, lightweight, if no system config tool due to it being tedious to set up networking
+- Team Robot communication – only one robot connects to GameController and then sends information to the other robots
```

Comment:
This line was an idea to help with the GameController issues. Will need to look into the rules. Not sure if it'd work anyway since it requires networking anway. We are already looking into audio, a board is being made and there is a speech recognition WIL project.